client/account-recovery/controller.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import DocumentHead from 'components/data/document-head';
+import LostPassword from 'account-recovery/lost-password';
+
+const ANALYTICS_PAGE_TITLE = 'Account Recovery';
+
+export default {
+	lostPassword( context ) {
+		const basePath = context.path;
+
+		analytics.pageView.record( basePath, `${ ANALYTICS_PAGE_TITLE } > Lost Password` );
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/8955/commits/1e728a3d37f46c00de24a6ba792e4378c06cb5c0